### ISSUE TITLE
Error while using the pretrained Synth 90K model

### ISSUE THREAD

**Issue by saurabhzuve** [da03/Attention-OCR#53]
Hey please help,
I am trying to run the code on the pretrained model as provided by you. But I get the following error when I run:
python src/launcher.py --phase=test --visualize --data-path=my_img/image_path.txt --data-base-dir=my_img --log-path=log.txt --load-model --model-dir=model --output-dir=results

Error:
NotFoundError (see above for traceback): Tensor name "bidirectional_rnn/bw/basic
_lstm_cell/bias" not found in checkpoint files model\translate.ckpt-47200
         [[Node: save/RestoreV2_1 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:l
ocalhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_1/tensor_n
ames, save/RestoreV2_1/shape_and_slices)]]

I am running this on Windows using Tensorflow 1.2.1

Please help

**Comment by ArulMuruganChinnakannan** [comment]
I face the same Problem Please help

I am running this on Windows using Tensorflow 1.3
![image](https://user-images.githubusercontent.com/5483420/31228442-307e6986-a9fb-11e7-810a-d410220c5db0.png)

![image](https://user-images.githubusercontent.com/5483420/31228236-6788ad98-a9fa-11e7-87a4-1f2e7ecccc0a.png)

**Comment by cbzeng110** [comment]
I face the same problem. But when use the model trained by myself, this problem is solved

**Comment by pribadihcr** [comment]
Hi @cbzeng110,

could you give me a link to download your model?. how the accuracy?

**Comment by cbzeng110** [comment]
u can train you own model with the instruction by the author

**Comment by balajiwix** [comment]
When i am doing the training , i am getting the errors

 2018-05-16 16:41:13,307 PIL.PngImagePlugin DEBUG    STREAM b'tIME' 41 7
2018-05-16 16:41:13,307 PIL.PngImagePlugin DEBUG    b'tIME' 41 7 (unknown)
2018-05-16 16:41:13,307 PIL.PngImagePlugin DEBUG    STREAM b'IDAT' 60 8192
2018-05-16 16:41:13,308 PIL.PngImagePlugin DEBUG    STREAM b'IHDR' 16 13
2018-05-16 16:41:13,308 PIL.PngImagePlugin DEBUG    STREAM b'tIME' 41 7
2018-05-16 16:41:13,308 PIL.PngImagePlugin DEBUG    b'tIME' 41 7 (unknown)
2018-05-16 16:41:13,309 PIL.PngImagePlugin DEBUG    STREAM b'IDAT' 60 1699
2018-05-16 16:41:13,309 PIL.PngImagePlugin DEBUG    STREAM b'IHDR' 16 13
2018-05-16 16:41:13,309 PIL.PngImagePlugin DEBUG    STREAM b'tIME' 41 7
2018-05-16 16:41:13,309 PIL.PngImagePlugin DEBUG    b'tIME' 41 7 (unknown)
2018-05-16 16:41:13,309 PIL.PngImagePlugin DEBUG    STREAM b'IDAT' 60 1871
2018-05-16 16:41:13,310 PIL.PngImagePlugin DEBUG    STREAM b'IHDR' 16 13
2018-05-16 16:41:13,310 PIL.PngImagePlugin DEBUG    STREAM b'tIME' 41 7
2018-05-16 16:41:13,310 PIL.PngImagePlugin DEBUG    b'tIME' 41 7 (unknown)
2018-05-16 16:41:13,310 PIL.PngImagePlugin DEBUG    STREAM b'IDAT' 60 8192
2018-05-16 16:41:13,310 PIL.PngImagePlugin DEBUG    STREAM b'IHDR' 16 13
2018-05-16 16:41:13,311 PIL.PngImagePlugin DEBUG    STREAM b'tIME' 41 7
